mobile/docs/iOS_BUILD.md

```diff
@@ -0,0 +1,155 @@
+# iOS Build Guide
+
+## Issue Diagnosis: module 'flutter_secure_storage' not found
+
+### Root Cause
+This error occurs because CocoaPods dependencies have not been installed. `flutter_secure_storage` is a Flutter plugin that requires native platform support, and its iOS native code must be installed via CocoaPods.
+
+### Solution
+
+#### First-time Setup or After Dependency Updates
+```bash
+# 1. Get Flutter dependencies
+flutter pub get
+
+# 2. Navigate to iOS directory and install CocoaPods dependencies
+cd ios
+pod install
```

Comment:
Just created [this issue](https://github.com/we-promise/sure/issues/492) when I came across this piece. I think it'd be awesome if we could ensure easily that the right version of CocoaPods is present in every environment.